jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -155,6 +155,7 @@
         <projectService serviceImplementation="software.aws.toolkits.jetbrains.settings.LambdaSettings"/>
 
         <moduleService serviceImplementation="software.aws.toolkits.jetbrains.settings.DeploySettings"/>
+        <projectService serviceImplementation="software.aws.toolkits.jetbrains.settings.UpdateLambdaSettings"/>
```

Comment:
`DeploySettings` is a module service - should this be also?

Reply:
According to Austin `ModuleService is an antipattern now`, and it would not make any sense anyway since that maps it backwards.